discus/api/views.py

```diff
@@ -1,10 +1,18 @@
 from flask import Flask
+from flask import request
```

Comment:
This import should probably be merged with the one above it. Something like `from flask import Flask, request`